starter-code/css/main.css

```diff
@@ -1,8 +1,168 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+
+html, body {
+    margin:0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Varela Round', sans-serif;
+    /*line-height: 3.5em;*/
+    color:#1A1A1A;
+}
+
+.nav-bar {
+    position: fixed;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color:#FFF;
+    width: 100%;
+
+}
+
+.nav-bar ul{
+    width: 40%;
+    margin-right: 20px;
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+}
+
+.nav-bar ul li{
+    padding: 2px;
+    font-size: 1em;
+}
+
+.nav-bar div img {
+    width: 125px;
+}
+
+.conteiner {
+    background-image:url("./../images/landing.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: transparent;
+    height: 650px;
+    padding-top: 50px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+}
+
+h1 {
+    font-weight: lighter;
+    font-size: 3em;
+    text-align: center;
+    color:#FFF;
+    letter-spacing: 4px;
+}
+
+h2 {
+    font-weight: lighter;
+    font-size: 1.25em;
+    text-align: center;
+    color:#FFF;
+    line-height: 1%;
+    letter-spacing: 0.6px;
+}
+
+#section1 {
+    width: 100%;
+    text-align: center;
+}
+
+#section1 h3 {
+    border-bottom: 3px solid #00B172;
+    display: inline-block;
+}
+
+.icons {
+    display: flex;
+    justify-content: space-around;
+}
+
+.service-box {
+    width: 200px;
+    height: 200px;
```

Comment:
Having a fixed width and height here causes your h4 inside the service box to overlap because it takes more space that the fixed height you have defined.